test/wrappedNodeFetch.test.ts

```diff
@@ -0,0 +1,118 @@
+import { wrappedNodeFetch } from '../integrations/octokit/require';
+import { Response } from 'node-fetch';
+import { createExecutionContext, getExecutionContext} from '../src/context';
+import { HTTP } from '../src/keploy';
+
+describe('wrappedNodeFetch', () => {
+  it('should call fetch function with correct arguments in record mode', async () => {
+    const mockFetch = jest.fn().mockResolvedValueOnce(new Response());
```

Comment:
Since, outputs are recorded when end event is triggered for response of fetch. We dont need to mock fetch here. We can use the actual fetch of node-fetch